File: dotnet/SDL.DXA.Modules.CampaignContent/Provider/CampaignContentMarkup.cs
using System;
using System.IO;

namespace SDL.DXA.Modules.CampaignContent.Provider
{
    /// <summary>
    /// Campaign Content Markup
    /// </summary>
    public class CampaignContentMarkup
    {
        public string HeaderHtml { get; set; }
        public string MainHtml { get; set; }
        public string FooterHtml { get; set; }
        public DateTime LastModified { get; set; }
        public string ZipFileName { get; set; }
    }
}

File: java/campaigncontent-dxa-module/src/main/java/com/sdl/dxa/modules/campaigncontent/provider/CampaignContentMarkup.java
package com.sdl.dxa.modules.campaigncontent.provider;

import lombok.Getter;
import lombok.Setter;

/**
 * Holder for the different markup in the campaign (main, header, footer)
 */
@Getter
@Setter
public class CampaignContentMarkup {

    private String headerHtml;
    private String mainHtml;
    private String footerHtml;
    long lastModified;
}
